pages/index.js

```diff
@@ -1,7 +1,34 @@
+import data from "@/data/tasks.json";
+import { useState } from "react";
+import { TaskCard } from "@/components/TaskCard";
 export default function HomePage() {
+  const [visible, setVisible] = useState(true);
+  console.log(data);
```

Comment:
Remove `console.logs` before you submit your PR for review.

Reply:
OK